### ISSUE TITLE
Plugin autostarting music

### ISSUE THREAD

**Issue by DJkefo** [surfbryce/beautiful-lyrics#170]
Once again, when this plugin is loaded, the currently selected music will play on startup of spotify. I don't want it to do that. Id on't get notified on any changes to this thread anywhere, I hope I'll see your answer fast enough so I can send any changes before this ticket gets closed.

**Comment by surfbryce** [comment]
@DJkefo I don't close tickets unless people don't respond to them within a certain amount of time or if I've fixed the bug and there's also no response from the person who made the issue.

It is your responsibility just as much as mine to pay attention to the issue at hand and if I respond and you expect a fix then you must also be willing to respond likewise. Obviously it can be a day-delay or whatever but what matters is that you check in to see if I've made a response so you can also respond to it.

Can you send the Spotify version you are on and same with Spicetify?

**Comment by surfbryce** [comment]
@DJkefo It didn't come across as harsh at all and I wasn't thinking you were accusing me of anything. I was just stating my perspective on the whole lack of notification thing.

So we are both on the same Spotify but you are a few versions behind (I'm on 2.24.2). What I will say is this: Spicetify made an update that changed a few things and I updated the extension to accommodate that a while back. Everybody who was having the issue you are reporting said it was fixed, so I'm thinking it's your Spicetify version being out of date but you let me know if that fixes it because as far as bug-reports go this issue has been fixed for a while now.
